cfuncs.py

```diff
@@ -0,0 +1,67 @@
+import ctypes
+from ctypes.wintypes import BOOL
+from ctypes.wintypes import DWORD
+from ctypes.wintypes import HANDLE
+from ctypes.wintypes import LPVOID
+from ctypes.wintypes import LPCVOID
+from pywinauto.win32structures import SECURITY_ATTRIBUTES
+from pywinauto import win32defines
+
+'''
+    Some functions already exists in pywinauto, but for correctly
+    injector work needs full redefinition with ctypes compatible
```

Comment:
This doc string could be re-written simpler and using triple double quotes (`"""`) according to PEP8. Reference to `pywinauto.remote_memory_block` could be more precise for such a TODO.